jdk-1.7-parent/openlayers3-parent/openlayers3/src/main/java/org/wicketstuff/openlayers3/DefaultOpenLayersMap.java

```diff
@@ -27,12 +28,12 @@ public DefaultOpenLayersMap(final String id, final IModel<Map> model) {
 
     @Override
     public void renderHead(final IHeaderResponse response) {
-        response.render(CssHeaderItem.forUrl("http://ol3js.org/en/master/css/ol.css"));
```

Comment:
You can also use `CssHeaderItem.forUrl("//ol3js.org/en/master/css/ol.css")` to make it scheme relative.